application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -278,6 +278,146 @@ public function import_survey($sSessionKey, $sImportData, $sImportDataType, $sNe
                     return array('status' => self::INVALID_SESSION_KEY);
         }
     }
+    
+	/**
+     * RPC Routine to export a survey archive (LSA).
+     *
+     * @access public
+     * @param string $sSessionKey Auth credentials
+     * @param int $iSurveyID_org Id of the survey
+     * @return string|array in case of success : Base64 encoded string of the .lsa file. On failure array with error information.
+     * */
+    public function export_survey_archive ($sSessionKey, $iSurveyID_org)
+    {
+        $iSurveyID = (int) $iSurveyID_org;
+        if (!$this->_checkSessionKey($sSessionKey)) {
+            return array('status' => 'Invalid session key');
+        }
+        $aData['bFailed'] = false; // Put a var for continue
+        if (!$iSurveyID) {
+            $aData['sErrorMessage'] = "No survey ID has been provided. Cannot export survey";
+            $aData['bFailed'] = true;
+        } elseif (!Survey::model()->findByPk($iSurveyID)) {
+            $aData['sErrorMessage'] = "Invalid survey ID";
+            $aData['bFailed'] = true;
+        } elseif (!Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'export') && !Permission::model()->hasSurveyPermission($iSurveyID, 'surveycontent', 'export')) {
+            $aData['sErrorMessage'] = "You don't have sufficient permissions.";
+            $aData['bFailed'] = true;
+        } else {
+            $aExcludes = array();
+            $aExcludes['dates'] = true;
+            $btranslinksfields = true;
+            Yii::app()->loadHelper('export');
+
+            /* START CODE DUPLICATION
```

Comment:
Why not include and update application/controllers/admin/export.php:_exportarchive function ?
To set it as public ?

or create a public function in export ?